### PR TITLE
[Technical Support] LPS-86932 fix ddPlugins node and contsrain css classes for RepeatableSortableList

### PR DESCRIPTION
**Pull request by natocesarrego**
Hey @rafaprax,

I just reviewed this pull request and it looks good to me, however I still think that would be nice a front-end engineer take a look on it, like @clovisdasilvaneto and @alinedoleron.

Best.

### CODE REVIEW

**Comment by rafaprax** [comment]
Hey @clovisdasilvaneto and @alinedoleron, Can you help me with this review? Thx

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js [hunk 1]
```diff
@@ -3119,14 +3119,14 @@ AUI.add(
 								ddPlugins.push(
 									{
 										cfg: {
-											constrain: '.lfr-form-content'
+											constrain: '.lfr-ddm-container'
```
Comment (clovisdasilvaneto) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: @natocesarrego, Is this class stylized in the forms css?

**Comment by rafaprax** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by rafaprax** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/64860. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js (hunk 1 above)
Comment (rafaprax) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js]: HEy @clovisdasilvaneto I checked and this class is not stylized in the forms css. Thx

**Pull request closed without merging**